graph/pull.go

```diff
@@ -428,7 +428,7 @@ func (s *TagStore) pullV2Tag(eng *engine.Engine, r *registry.Session, out io.Wri
 	}
 
 	if verified {
-		out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified"))
+		out.Write(sf.FormatStatus(repoInfo.CanonicalName+":"+tag, "The image you are pulling has been verified.\nThis is a tech preview, don't rely on it for security yet."))
```

Comment:
I actually think we should just remove this one, it is only verifying the manifest not the layers.